tests/post_training/test_quantize_conformance.py

```diff
@@ -280,6 +281,7 @@ def test_ptq_quantization(
         test_model_param = PTQ_TEST_CASES[test_case_name]
         maybe_skip_test_case(test_model_param, run_fp32_backend, run_torch_cuda_backend, batch_size)
         pipeline_cls = test_model_param["pipeline_cls"]
+        base_precision = test_model_param.get("base_precision", PrecisionType.FP32)
```

Comment:
use `param` to store `base_precision`. 
Like it done for `is_stateful`

https://github.com/openvinotoolkit/nncf/blob/develop/tests/post_training/model_scope.py#L58
https://github.com/openvinotoolkit/nncf/blob/develop/tests/post_training/pipelines/causal_language_model.py#L27